tests/test_hydra.py

```diff
@@ -1585,3 +1585,30 @@ def test_hydra_verbose_1897(tmpdir: Path, multirun: bool) -> None:
     if multirun:
         cmd += ["+a=1,2", "-m"]
     run_python_script(cmd)
+
+
+@mark.parametrize(
+    "multirun",
+    [False, True],
+)
+def test_hydra_resolver_in_output_dir(tmpdir: Path, multirun: bool) -> None:
+    from hydra import __version__
+
+    output_dir = str(tmpdir) + "${val}"
```

Comment:
What do you think about replacing `${val}` with `${hydra:runtime.version}` here? We could then get rid of the config file (to make the test simpler and easier to understand for future maintainers).